Data/ByteString/Builder/Prim.hs

```diff
@@ -575,13 +575,7 @@ primBounded w x =
 -- TODO: The same rules for 'putBuilder (..) >> putBuilder (..)'
 
 -- | Create a 'Builder' that encodes a list of values consecutively using a
--- 'BoundedPrim' for each element. This function is more efficient than the
--- canonical
---
--- > filter p =
--- >  B.toLazyByteString .
--- >  E.encodeLazyByteStringWithF (E.ifF p E.word8) E.emptyF)
--- >
```

Comment:
I removed this because it has nothing to do with lists. A similar, and up-to-date example is already in the comment of`primMapLazyByteStringBounded`